main.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     // Load initial document
     var defaultHTML = brackets.getModule("text!filesystem/impls/filer/lib/default.html");
+    var defaultCSS  = brackets.getModule("text!filesystem/impls/filer/lib/style.css");
+    var defaultJS   = brackets.getModule("text!filesystem/impls/filer/lib/script.js");
```

Comment:
You can't use `require` here because you need a path into the `src/` dir.  `brackets.getModule` *is* `require` with the path setup properly.